vtStor/DataCommandStructure.h

```diff
@@ -0,0 +1,41 @@
+/*
+<License>
+Copyright 2016 Virtium Technology
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http ://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+</License>
+*/
+
+#ifndef __DataCommandStructure_h__
+#define __DataCommandStructure_h__
+#pragma once
+
+#include <vector>
+
+#include "BasicTypes.h"
+
+namespace vtStor
+{
+    struct sLBARangeEntry
+    {
+        U64 LBA;
+        U16 SectorCount;
+    };
+
+    struct sLBARangeEntries
```

Comment:
It use for same format with LBARangeEntriesManage (List type) object which removed. I will remove this struct